app/src/main/java/org/listenbrainz/android/presentation/features/brainzplayer/ui/BrainzPlayerBackDropScreen.kt

```diff
@@ -249,7 +249,7 @@ fun PlayerScreen(
                                 }
                             }
                         },
-                    tint = if (!listenLiked) Color.Red else Color.Black
+                    tint = Color.Red
```

Comment:
We shouldn't be doing this. Rather that this, change the else condition to pick the onSurface color I think

Reply:
yeah, i think it`s a good idea, thanks for suggesting